crates/datasources/Cargo.toml

```diff
@@ -58,6 +58,7 @@ webpki-roots = "0.25.1"
 dashmap = "5.5.0"
 calamine = { version = "0.22.1", features = ["dates"] }
 tiberius = { version = "0.12.2", default-features = false, features = ["tds73", "rustls", "chrono"] }
+lance = { git = "https://github.com/universalmind303/lance", rev = "81158eb540ff88ab5b4fce3a1170447760137412" }
```

Comment:
What's in your fork that's not yet upstream?

Reply:
currently only this PR. 
https://github.com/lancedb/lance/pull/1608

Reply:
It's currently approved, but their CI seems to be failing due to unrelated issues. 

I'm totally fine waiting for the PR to get merged before merging this.

Reply:
I think it's fine, we can open up a separate pr once it's merged.